### ISSUE TITLE
Regarding interpreting mAP values

### ISSUE THREAD

**Issue by sudharavali** [AlexeyAB/darknet#3239]
Hi @AlexeyAB, 

Can you tell me what it means when the mAP value on the test dataset is greater than validation dataset ? I am not able to understand why this happens, mAP for test is 99.5% and validation is 97%. Please let me know when this happens !

**Comment by AlexeyAB** [comment]
@sudharavali Hi,

Did you train on Training dataset, and then check mAP on Test and Validation sets? 
How many images are in the Train, Valid and Test?

> Can you tell me what it means when the mAP value on the **test dataset is greater** than validation dataset ? 

As I see `test is 99.5%` **lower** than validation is 97%.

> I am not able to understand why this happens, mAP for test is 99.5% and validation is 97%.

It is just a random fluctuation.

**Comment by sudharavali** [comment]
> Did you train on Training dataset, and then check mAP on Test and Validation sets?

Yes, thats what I did. 

I trained on 5000 images, validation dataset is about 560 images, test dataset is 1000 images. 

> It is just a random fluctuation.

Does this mean that the accuracy is pretty good on test and validation sets and there is nothing more that we can interpret from this ?

**Comment by AlexeyAB** [comment]
Also may be we can say that there is a small number of images.
